### DISCUSSION TITLE
How can I enter new tokens for the same car?

### DISCUSSION THREAD

**Discussion by XtremeLead** [teslamate-org/teslamate#4354] [General]
I have a running teslamate installation. I need to update my tokens, because I have a new account for the same car. How can I update my tokens? I want to keep all my history so I don't want to start over

**Comment by cwanja** [comment]
Log out of TeslaMate and login with new Tesla account.

**Comment by XtremeLead** [comment]
omg...seriously?? Totally missed the sign out link. Thanks. That worked
